packages/controller-utils/src/constants.ts

```diff
@@ -116,3 +116,9 @@ export const NETWORK_ID_TO_ETHERS_NETWORK_NAME_MAP: Record<
   [NetworkId.sepolia]: NetworkType.sepolia,
   [NetworkId.mainnet]: NetworkType.mainnet,
 };
+
+export enum Source {
```

Comment:
Since this type is only used within `assets-controllers`, can it live inside there? Also, I wonder if this name is too generic — I can see it clashing with something else in the future. What do you think about `NftSource`?

Reply:
Sure! I was thinking that it might be useful for other controllers (particularly for metrics). But yeah I suppose those other controllers that would use it would probably be in the `assets-controllers` package. Do you think we should create a shared constants file for that package? And regarding the name I was thinking we keep it generic again because it can probably be used for other non-nft specific cases in the future. But perhaps we can just rename it when it gets used as such?

Reply:
did something like this here: https://github.com/MetaMask/core/pull/1417/commits/56466f2aee5f6a2fa3882e420f349fa656642397

Reply:
Sorry I was caught up in meetings today. I think that's better!